packages/flutter/test/scheduler/scheduler_test.dart

```diff
@@ -122,7 +120,8 @@ void main() {
         createTimer: (Zone self, ZoneDelegate parent, Zone zone, Duration duration, void f()) {
           // Don't actually run the tasks, just record that it was scheduled.
           timerQueueTasks.add(f);
-          return null;
+          // TODO(darrenaustin): find a cleaner way of having an empty Timer returned here
+          return null as Timer;
```

Comment:
Any ideas on how to make this less gross? I tried to just have it return a `Timer` with an empty callback, but that lead to an infinite loop in the scheduler.

Should we make the `CreateTimerHandler` return a `Timer?` instead? If returning null works here, something must be handling a null values for the return value somewhere.

Reply:
Here's my suggested solution to it:

```dart
// ...

return DummyTimer();

// ...

class DummyTimer implements Timer {
  @override
  void cancel() {}

  @override
  bool get isActive => false;

  @override
  int get tick => 0;
}
```

Reply:
Thanks! That does indeed work. Very odd that a default empty `Timer` didn't do the same thing.